From f2e8a20a96d35dfe4e7276d9202766a780e240a8 Mon Sep 17 00:00:00 2001
From: Kha Tran <kha.tran.px@rvc.renesas.com>
Date: Thu, 25 Jan 2018 10:29:49 +0700
Subject: [PATCH 323/557] ARM: DTS: r8a7744: Add Image Extraction Direct Memory
 Access Controller (tddmac module) node to device tree

This patch support Renesas tddmac kernel module probe.

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: Kha Tran <kha.tran.px@rvc.renesas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
---
 arch/arm/boot/dts/r8a7744.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7744.dtsi b/arch/arm/boot/dts/r8a7744.dtsi
index 9a77e7d..72355fc 100644
--- a/arch/arm/boot/dts/r8a7744.dtsi
+++ b/arch/arm/boot/dts/r8a7744.dtsi
@@ -1520,6 +1520,14 @@
 			renesas,#ch = <0>;
 		};
 
+		tddmac@0xfea00000 {
+			compatible = "renesas,vspm-tddmac","renesas,vspm";
+			reg = <0 0xfea00000 0 0x200>;
+			interrupts = <0 285 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&mstp1_clks R8A7744_CLK_2DDMAC>;
+			power-domains = <&sysc R8A7744_PD_ALWAYS_ON>;
+		};
+
 		vspm@fe928000 {
 			compatible = "renesas,vspm-vsps","renesas-vspm";
 			reg = <0 0xfe928000 0 0x7404>;
-- 
2.7.4

